### PR TITLE
Main page prototype

### PR DESCRIPTION
**Pull request by TsarN**

### CODE REVIEW

File: coffee-chats/src/main/webapp/src/components/ConnectBackCard.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+          </Typography>
+        </CardContent>
+        <CardActions>
+          <Button size="small">More</Button>
+        </CardActions>
+      </Card>
+  )
+}
+
+export default ConnectBackCard;
```
Comment (PrabSG) [inline coffee-chats/src/main/webapp/src/components/ConnectBackCard.tsx:29]: Google Styleguide says to not use default exports unfortunately, despite React's tendency to use them. Perhaps  @DarthGandalf can clarify whether to follow the Google JS styleguide, or stick with the React convention?

File: coffee-chats/src/main/webapp/src/components/ConnectBackCard.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  return (
+      <Card>
+        <CardContent>
+          <Typography variant="h5" gutterBottom>
+            Connect Back
+          </Typography>
+          <Typography color="textSecondary" gutterBottom>
+            You've recently chatted
+            about {humanReadableJoin(props.tags)} {}
+            with {}
```
Comment (DarthGandalf) [inline coffee-chats/src/main/webapp/src/components/ConnectBackCard.tsx:16]: What do empty {} mean in this syntax?

File: coffee-chats/src/main/webapp/src/components/ConnectBackCard.tsx (hunk 1 above)
Comment (DarthGandalf) [reply coffee-chats/src/main/webapp/src/components/ConnectBackCard.tsx:29]: It should be easy enough to follow Google guide by adding {} around the name here and on import side

File: coffee-chats/src/main/webapp/src/components/ConnectBackCard.tsx (hunk 2 above)
Comment (TsarN) [reply coffee-chats/src/main/webapp/src/components/ConnectBackCard.tsx:16]: Otherwise there's no space around tags, because JSX transpiler strips spaces

Comment (DarthGandalf) [reply coffee-chats/src/main/webapp/src/components/ConnectBackCard.tsx:16]: But there are no tags on the previous line

**Pull request merged**